packages/core-js/internals/fix-regexp-well-known-symbol-logic.js

```diff
@@ -5,24 +5,30 @@ var fails = require('../internals/fails');
 var requireObjectCoercible = require('../internals/require-object-coercible');
 var wellKnownSymbol = require('../internals/well-known-symbol');
 
-module.exports = function (KEY, length, exec) {
+module.exports = function (KEY, length, exec, sham) {
   var SYMBOL = wellKnownSymbol(KEY);
   var methods = exec(requireObjectCoercible, SYMBOL, ''[KEY]);
   var stringMethod = methods[0];
   var regexMethod = methods[1];
   if (fails(function () {
+    var execCalled = false;
+    var re = /a/;
+    re.exec = function () { execCalled = true; return null; };
+    return re[SYMBOL] && (re[SYMBOL](''), execCalled);
```

Comment:
It should return a negative result for the existent feature. Use, for example,
```js
re[SYMBOL]('');
return !execCalled;
```

Reply:
It's just an example. The main idea of the comment - the result should be inverted.